src/world/Network/Handlers/ActionHandler.cpp

```diff
@@ -38,7 +38,7 @@ void Sapphire::Network::GameConnection::actionHandler( FrameworkPtr pFw,
   {
     default:
     {
-      player.sendDebug( "Skill type {0} not supported. Defaulting to normal action", type );
+      player.sendDebug( "Skill type {0} not supported. Defaulting to normal action", type ); break;
```

Comment:
Why is there a break here? the message just before it indicates that this is intended behaviour